### DISCUSSION TITLE
Why is there no `tokio-sink`?

### DISCUSSION THREAD

**Discussion by byte-sourcerer** [tokio-rs/tokio#5470] [Q&A] [answered]
There is `tokio-stream` out there, but why is there no `tokio-sink`?

[Similar question in stackoverflow](https://stackoverflow.com/questions/74457576/tokio-sender-is-not-sink).

**Comment by Darksonn** [comment, marked as answer]
There are some `Sink` implementations in `tokio-util`, e.g. there's `PollSender` for the mspc channel. There isn't one for the broadcast channel.

In general, streams are used much more often than sinks, which is why they have a dedicated crate.
